src/core/processing/qgsprocessingcontext.h

```diff
@@ -794,6 +794,13 @@ class CORE_EXPORT QgsProcessingContext
      */
     QgsProcessingModelResult &modelResult() SIP_SKIP { return mModelResult; }
 
+    /**
+     * Clears model results previously populated when the context was used to run a model algorithm.
+     *
+     * \since QGIS 3.42
```

Comment:
Since the PR must be backported to release-3_40 branch, shouldn't this method be "since QGIS 3.40"?

Reply:
Well, I don't think so (But not 100% sure). IIRC, that's the way we deal with such scenario, because the modification wouldn't be effective in 3.4.**0** only in 3.40.**3** and that could lead to misunderstanding for people who would read the documentation.

Reply:
@troopa81 is correct -- the convention we use for this is just to tag with the newer version, so that there's no risk of someone incorrectly assuming it's available in an older release